src/pages/workspaces/workspace/Data.js

```diff
@@ -81,7 +81,7 @@ const DataTypeButton = ({ selected, entityName, children, entityCount, iconName
 }
 
 const DataImportPlaceholder = () => {
-  return div({ style: { ...Style.navList.item(false), color: colors.dark(0.7) } }, [
+  return div({ style: { ...Style.navList.item(false), color: colors.dark(0.7), marginLeft: '0.5rem' } }, [
```

Comment:
The "Data import in Progress" icon and message were right up against the edge of the window, which looked bad.